src/Psalm/ErrorBaseline.php

```diff
@@ -261,6 +263,26 @@ function (string $extension) : string {
         $baselineDoc->appendChild($filesNode);
         $baselineDoc->formatOutput = true;
 
-        $fileProvider->setContents($baselineFile, $baselineDoc->saveXML());
+        $xml = preg_replace_callback(
+            '/<files (psalm-version="[^"]+") (?:php-version="(.+)">\n)/',
+            function (array $matches) : string {
+                return
+                    '<files' .
+                    "\n  " .
+                    $matches[1] .
```

Comment:
Right operand cannot be mixed
```
                    $matches[1] .
                    ^^^^^^^^^^^
```

Reply:
should be resolved by 7de56af, but is impacted by #1896